src/main/java/xyz/kyngs/herbot/handlers/command/commands/admin/PermissionCommand.java

```diff
@@ -0,0 +1,45 @@
+package xyz.kyngs.herbot.handlers.command.commands.admin;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import xyz.kyngs.herbot.HerBot;
+import xyz.kyngs.herbot.handlers.command.AbstractCommand;
+import xyz.kyngs.herbot.handlers.user.UserProfile;
+import xyz.kyngs.herbot.util.MemberUtil;
+
+import java.awt.*;
+
+public class PermissionCommand extends AbstractCommand {
+
+    public PermissionCommand(HerBot herBot, String description) {
+        super(herBot, description, "permissions");
+        addArg(String.class, "give", "take");
+        addArg(String.class, "uživatel");
+        addArg(String.class, "oprávnění");
+    }
+
+    @Override
+    public void exec(User author, Guild guild, TextChannel channel, Message message, String[] args, UserProfile profile, GuildMessageReceivedEvent event) {
+        User user = MemberUtil.getUser(args[1], herBot.getJda());
```

Comment:
Could you please give names for args[0], args[1] and args[2]? I assume that args[0] is command, args[1] is user1 and args[2] is user2. It might improve readability.

Reply:
So when u use addArg for first time in the constructor, it basically documents arg[0] (says what it should be and gives it required type), when you use addArg second time, it does the same thing but for arg[1] etc.